src/style.css

```diff
@@ -1,3 +1,60 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  max-width: 1200px;
+  min-width: 1024px;
+}
+
+.header {
   margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px 0 50;
```

Comment:
The padding property in the .header class is missing a unit for the last value. It should be 'padding: 0 40px 0 50px;' to ensure consistent rendering across browsers.